db/schema.rb

```diff
@@ -33,10 +33,10 @@
     t.index ["user_id"], name: "index_alerts_on_user_id"
   end
 
-  create_table "articles", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
+  create_table "articles", id: :integer, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8", force: :cascade do |t|
```

Comment:
These changes to schema.rb shouldn't be included in the PR. Running migrations automatically regenerates schema.rb and the exact output can vary based on the local environment, so people often include such changes by accident.